style-guide.md

```diff
@@ -50,116 +50,133 @@ This section of the document provides guidelines on best practices, language usa
 
 #### Numbers
 
-- Spell out numbers as words for numbers zero through nine
+- Spell out numbers as words for numbers zero through nine.
 
 #### Quotes
 
-- Use double quotes in regular text
-- Commas and periods go inside quotation marks
-- Single quotes should only be used in code examples, depending on the guidelines in the [Code Formatting By Language](#code-formatting-by-language) section
+- Use double quotes in regular text.
+- Commas and periods go inside quotation marks.
+- Single quotes should only be used in code examples, depending on the guidelines in the [Code Formatting By Language](#code-formatting-by-language) section.
 
 #### Capitalization
 
-- Use Chicago title-style capitalization for titles and headings, which capitalizes the first word plus all other significant words
-- Capitalize product names
-- Avoid unnecessary capitalization; before you capitalize a word, think about why and if it should be capitalized
+- Use Chicago title-style capitalization for titles and headings, which capitalizes the first word plus all other significant words.
+- Capitalize product names.
+- Avoid unnecessary capitalization; before you capitalize a word, think about why and if it should be capitalized.
 
 ### Table Formatting
 
-- Use tables to represent sets of related pieces of data in a structured way
-- Table headers and values should be centered
-- Tables should be formatted. You can use a tool to format the tables, like the [Markdown Table Formatter VSCode extension](https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter)
+- Use tables to represent sets of related pieces of data in a structured way.
+- Table headers and values should be centered.
+- Tables should be formatted. You can use a tool to format the tables, like the [Markdown Table Formatter VSCode extension](https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter).
 
 ### List Formatting
 
-- Use ordered (numbered) lists for a sequence of steps
-- Use unordered (bulleted) lists for items that are non-sequential and can be read or completed in any order
-- For description lists, use the following formatting: `**term** - description`
-  - Put the term in bold
-  - Use a dash (`-`) between the term and the description
-  - Do not capitalize the first word in the description, unless it's an acronym or product name that should be capitalized
-- Do not add punctuation at the end of each list item
+- Use ordered (numbered) lists for a sequence of steps.
+- Use unordered (bulleted) lists for items that are non-sequential and can be read or completed in any order.
+- For description lists, use the following formatting: `**Term**: Description.`
+  - Put the term in bold.
+  - Capitalize the term.
+  - Use a colon (`:`) between the term and the description.
+  - Capitalize the first word of the description, unless the first word is a code element (e.g., a function name, file name, or variable).
```

Comment:
This is quite specific, and it might imply that things that aren't code elements should still be capitalized when maybe they don't need to be. For example, how kluster never uses a capital K. I'd just say "use sentence case" instead.